src/pages/[[...slug]].tsx

```diff
@@ -33,19 +25,31 @@ import {
 } from '@/lib/drupal/staticProps'
 import { StaticPropsResource } from '@/lib/drupal/staticProps'
 import { FormattedPageResource } from '@/data/queries'
-import { LayoutProps } from '@/templates/layouts/wrapper'
+
```

Comment:
No functionality changes here. I just alphabetized and grouped imports a bit for easier reading.